lang/cs/BUILD.md

```diff
@@ -77,29 +83,3 @@ modify `test_script` section as follows:
 
 `nuget restore` is necessary to install `xunit.runner.console` package, which by default is installed in `TestRunner.proj`.
 
-Instructions on Building and Testing REEF .NET from Scratch
```

Comment:
This effectively reverts the change done in #1226. @EstusDev Could you please take a look to see whether the issues you've experienced when building REEF.NET are still captured in the new version of the documentation?